aperture_properties.py

```diff
@@ -144,8 +144,6 @@ def MetalFracStar(self):
     get_angular_momentum,
     get_angular_momentum_and_kappa_corot,
     get_vmax,
-    get_inertia_tensor,
-    get_reduced_inertia_tensor,
```

Comment:
This is removed because we will not compute the inertia tensor for spherical apertures, right?

Reply:
Yep